iceoryx_examples/icedelivery_in_c/ice_c_publisher.c

```diff
@@ -77,6 +77,7 @@ void sending()
 int main()
 {
     signal(SIGINT, sigHandler);
+    signal(SIGTERM, sigHandler);
```

Comment:
okay, this is C code. Maybe we need a issue or extend #542 to replace these with `sigaction`. Could also be just a free function like `registerSignalHandler` but without the restoring of the previous signal handler

Reply:
I added this as a todo